src/js/actions/libraries.js

```diff
@@ -806,12 +812,9 @@ define(function (require, exports) {
                         // So, although the command does not resolve until the placement has finished,
                         // it does resolve before the layer model has been updated because the event
                         // has not yet been received. For more details see issue #2177.
-                        var placeElementPromise = descriptor.playObject(placeObj, { synchronous: false })
-                            .bind(this)
-                            .finally(function () {
-                                return this.transfer(policyActions.removeKeyboardPolicies, policyID, true);
-                            });
-                            
+                        return descriptor.playObject(placeObj, { synchronous: false });
```

Comment:
I think you should also catch errors here, and restorePolicies in that case.  

Related question: after the user places the object (and while policies are suspended) is it possible for them to "cancel" the operation (via ESC?)?  I'm trying to think of ways that we could initiate the placeObject flow, but then NOT end up getting an event back from photoshop.